examples/url-shotener/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"github.com/dicedb/dicedb-go" // DiceDB Go SDK
+)
+
+type URL struct {
+	ID       string `json:"id"`
+	LongURL  string `json:"long_url"`
+	ShortURL string `json:"short_url"`
+}
+
+var db *dicedb.Client
+
+// Initialize DiceDB connection
+func init() {
+	db = dicedb.NewClient(&dicedb.Options{
+		Addr: "localhost:7379",
+	})
+}
+
+// Creates a short URL from a given long URL
+func CreateShortURL(c *gin.Context) {
+	var requestBody URL
+	if err := c.ShouldBindJSON(&requestBody); err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": "Invalid request"})
+		return
+	}
+
+	// Generate unique short ID and construct the short URL
+	shortID := uuid.New().String()[:8]
+	requestBody.ID = shortID
+	requestBody.ShortURL = "http://localhost:8080/" + shortID
```

Comment:
Do we need to store `ID` and `ShortURL` in the struct?
Having just `LongURL` is sufficient?

The mapping b/w `ID` and `LongURL` is anyway stored in the DiceDB.